.github/workflows/ci.yml

```diff
@@ -3,6 +3,8 @@ name: Node.js CI
 on:
   pull_request:
     branches: ['**']
+  merge_group:
```

Comment:
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#merge_group